src/ui/components/ProtocolViewer.tsx

```diff
@@ -174,6 +175,25 @@ function ProtocolRequestDetail({
       reportBugLink = `${BACKEND_GITHUB_REPO_BASE_URL}/issues/new?body=${encodeURIComponent(
         body
       )}&title=${encodeURIComponent(title)}&labels=bug,bug-report`;
+
+      const qs = encodeURIComponent(
+        JSON.stringify({
+          time_range: 600,
+          granularity: 0,
+          breakdowns: ["trace.trace_id"],
+          calculations: [{ op: "COUNT" }],
+          filters: [
+            { column: "name", op: "starts-with", value: "replayprotocol" },
+            { column: "replay.session.id", op: "=", value: window.sessionId },
+            { column: "rpc.replayprotocol.command_id", op: "=", value: request.id },
+          ],
+          filter_combination: "AND",
+          orders: [{ op: "COUNT", order: "descending" }],
+          havings: [],
```

Comment:
Not sure how much of this is actually necessary, but honeycomb includes it in their links, so.